src/EditorFeatures/Core/Implementation/Formatting/FormatCommandHandler.FormatDocument.cs

```diff
@@ -53,12 +56,14 @@ private void ShowGoldBarForCodeCleanupConfigurationIfNeeded(Document document)
                               kind: InfoBarUI.UIKind.Button,
                               () =>
                               {
+                                  Logger.Log(FunctionId.CodeCleanupInfobar_ConfigureNow, KeyValueLogMessage.NoProperty);
                                   optionPageService.ShowFormattingOptionPage();
                               }),
                 new InfoBarUI(EditorFeaturesResources.Do_not_show_this_message_again,
                               kind: InfoBarUI.UIKind.Button,
                               () =>
                               {
+                                  Logger.Log(FunctionId.CodeCleanupInfobar_DoNotShow, KeyValueLogMessage.NoProperty);
```

Comment:
how about changing DoNotShow to NeverShowCodeCleanupInfoBarAgain since user could click "x" rather than "never show" button.

Reply:
Done